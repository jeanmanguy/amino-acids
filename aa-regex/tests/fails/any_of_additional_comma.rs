use aa_regex::*;

fn main() {
    let _wrong = any_of!(C,);
    let _wrong = any_of!(,);
}
