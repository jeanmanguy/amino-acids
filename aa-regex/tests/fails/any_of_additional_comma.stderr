error: expected identifier
 --> $DIR/any_of_additional_comma.rs:5:26
  |
5 |     let _wrong = any_of!(,);
  |                          ^
